e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -77,6 +77,60 @@
               "createOptions": ""
             }
           },
+          "twinTester1": {
+            "settings": {
+                "image": "<Container_Registry>/microsoft/azureiotedge-twin-tester:<Build.BuildNumber>-linux-<Architecture>",
+                "createOptions": ""
+            },
+            "type": "docker",
+            "version": "1.0",
+            "env": {
+                "TransportType": {
+                    "value": "amqp"
```

Comment:
why "amqp" and "mqtt"? do we need to test amqpws and mqttws?

Reply:
I tried to mirror load gen's configuration here. For longhaul, I notice that we test differently:
```
  transportType1: 'Amqp_Tcp_Only'
  transportType2: 'Amqp_WebSocket_Only'
  transportType3: 'Mqtt_Tcp_Only'
  transportType4: 'Mqtt_WebSocket_Only
```

Is there a reason we are not testing amqpWs and mqttWs in stresstest?

Reply:
For long haul, in general we didn't test all 4 protocols.  And this catches me because current implementation doesn't use text same as enum, e.g. 'Amqp_Tcp_Only' and 'Mqtt_Tcp_Only'.   It should be ok to test just 2 protocols for long haul.  We can talk about this during review meeting.

Reply:
Should this expect to be one of the 4 values in above comment? e.g. "Amqp_Tcp_Only".

Reply:
Actually amqp and mqtt is one of the value in TransportType enum; which provides the fallback to web socket of given protocol.  It is totally fine for now.  No change is needed.